lyluatex.lua

```diff
@@ -1198,7 +1211,6 @@ end
 function Score:run_lilypond(ly_code)
     if self:is_compiled() then return end
     mkdirs(dirname(self.output))
-    self:lilypond_version()
```

Comment:
I got slightly confused about this when reviewing my own staging area.

Am I right that this call was previously done exclusively to check for a successful LilyPond installation? If so it should be safe to remove it because that check is now done right at the entry of `Score:process()`.

@jperon

Reply:
In any case, as no return value is used here, and `lilypond_version()` has already been run at this time, there's certainly no need to keep it here.